spec/active_record/log_subscriber_spec.rb

```diff
@@ -10,10 +10,11 @@
 
     before do
       ActiveRecord::Base.logger = logger
+      @backup_disabled = Slavery.disabled
```

Comment:
This is irrelevant of the issue and should actually be a separate commit outside of this PR. It's always good to restore the value to what it was before, like we already do in other tests.